zpls_echograms/README.md

```diff
@@ -1,8 +1,20 @@
 **ZPLSC/G Echogram Code & Instructions**  
-Author: J. Kuo, jkuo@whoi.edu  
-8/14/2020  
+Author: J. Kuo, jkuo@whoi.edu; S. Petillo, spetillo@whoi.edu  
+8/19/2020  
 
-*Note:* The preset colorbar ranges are the same for every array. Feel free to change it in the code.  
+**Setup:**  
+1. Install Anaconda or Miniconda to manage Python-related packages and package versions  
+https://www.anaconda.com/products/individual  
+https://docs.conda.io/en/latest/miniconda.html  
+
+2. Create the conda environment, then install the conda environment to run with this code and it's echopype dependency (instead of using pip)  
```

Comment:
Change `it's` to `its` since you are using it as a possessive determiner. `it's` is a contraction of `it is`.

Reply:
The grammar police are after me! Hahaha. Can't believe I missed that one... Thanks! Fixed now.